File: Course_01/C Language/EstruturaRepetitiva/media_ponderada.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    double a, b, c, media;
    int n, i;

    printf("Quantos casos voce vai digitar?" );
    scanf("%d", &n);

    for (i = 0 ; i < n ; i++) {
        printf("Digite tres numeros:\n");
        scanf("%lf%lf%lf", &a, &b, &c);

        media = ((a * 2) + (b * 3) + (c * 5)) / 10;

        printf("MEDIA = %.1lf\n", media);
    }
    return 0;
}

File: Course_01/Java Language/Estrutura repetitiva/media_ponderada.java
package Repetitivo;

import java.util.Locale;
import java.util.Scanner;

public class media_ponderada {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n, i;
		double n1, n2, n3, media;
		
		System.out.print("Quantos casos você vai digitar? ");
		n = sc.nextInt();
		
		for (i = 0; i < n; i++) {
			System.out.println("digite tres numeros: ");
			n1 = sc.nextDouble();
			n2 = sc.nextDouble();
			n3 = sc.nextDouble();
			media = (n1 * 2 + n2 * 3 + n3 * 5) / 10;
			System.out.println("MEDIA = " + String.format("%.2f", media));
		}
		sc.close();
	}

}
